src/Core/Domain/Order/QueryResult/OrderPreviewProductDetail.php

```diff
@@ -66,6 +66,11 @@ class OrderPreviewProductDetail
      */
     private $location;
 
+    /**
+     * @var int
+     */
+    private $id;
```

Comment:
```suggestion
    private $productId;
```

Reply:
I'm not sure,  because the name of the class is already quite meaningful, it would be redundant. Ex.: In view : 
 `{$productDetail.productId}`  (product is redondant) instead of `{$productDetail.id}` (more elegant :))

Reply:
ping @jolelievre

Reply:
Yeah I thought about that that's why it was just a suggestion